fsl_simple_pipeline/src/realtime/recognizer.py

```
@@ -0,0 +1,225 @@
+import cv2
+import numpy as np
+import time
+from typing import Dict, List, Tuple, Optional, Callable
```

Ruff (0.8.2)

`typing.List` imported but unused

Remove unused import

(F401)